lib/postgrex/extensions/tsvector.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Postgrex.Extensions.TSVector do
+  @moduledoc false
+
+  import Postgrex.BinaryUtils, warn: false
+  use Bitwise
+  use Postgrex.BinaryExtension, [send: "tsvectorsend"]
+
+  def encode(_) do
+    quote location: :keep do
+      values when is_list(values) ->
+        encoded_tsvectors = unquote(__MODULE__).encode_tsvector(values)
+        <<byte_size(encoded_tsvectors)::int32, encoded_tsvectors::binary>>
+      other ->
+        raise ArgumentError,
+        Postgrex.Utils.encode_msg(other, "a list of tsvectors")
+    end
+  end
+
+  def decode(_) do
+    quote do
+      <<len::int32, value::binary-size(len)>> ->
+        <<nb_lexemes::int32, words::binary>> = value
+        unquote(__MODULE__).decode_tsvector_values(:binary.bin_to_list(words))
```

Comment:
I converted it to use the binary format everywhere. It should be an improvment.

Thanks!